std_msgs/Header header

int32 sentence_id

string file_path